cpp/test/Slice/errorDetection/WarningInvalidMetadata.ice

```diff
@@ -100,15 +100,15 @@ class P
 // Metadata is shared between forward declarations and definitions.
 // We allow duplicate metadata, but require that that metadata must be identical.
 
-["java:nonsense", "deprecated:hello", "protected"]
+["java:nonsense", "deprecated:hello"]
 class K;
```

Comment:
These tests weren't specifically for "protected", but metadata in general.
Since we can no longer use `protected` to check the behavior of duplicate metadata on multiple declarations,
we should do this instead:
```suggestion
["java:nonsense", "deprecated:hello", "amd"]
interface K;
```